src/RealtimeEffectPanel.cpp

```diff
@@ -290,8 +288,8 @@ namespace
       std::shared_ptr<RealtimeEffectState> mEffectState;
       std::shared_ptr<EffectSettingsAccess> mSettingsAccess;
 
-      ThemedButtonWrapper<wxButton>* mChangeButton{nullptr};
```

Comment:
The template `ThemedButtonWrapper` is now unused at commit "RealtimeEffectPanel wx*Button replaced with AButton" while the template `ThemedAButtonWrapper` is only used with AButton as the template argument.

I wonder if the templates are really needed then, but maybe you think there will be future uses.